em_stitch/lens_correction/mesh_and_solve_transform.py

```diff
@@ -17,13 +17,22 @@
 from bigfeta import jsongz
 import logging
 
+try:
+    # pandas unique is faster than numpy, use where appropriate
+    import pandas
```

Comment:
You don't want to just add pandas to `requirements.txt`?

Reply:
Comparitively it's not a huge speedup, so I don't necessarily think it's worth an added dependency.  Because they return things in different order, I am tempted to be explicit about it for testing purposes.